ionic2/app/pages/map/map.page.html

```diff
@@ -10,4 +10,7 @@
   </button>
 
   <poke-poi-card #poiCard></poke-poi-card>
+
+  <button id="artificial-mob-button"
```

Comment:
Let's leave it in for now, so that people can try it out. It's easy enough to remove later.